main.go

```diff
@@ -32,8 +35,13 @@ func main() {
 	router.HandleFunc("POST /posts", pc.CreatePost)
 	router.Handle("/", http.NotFoundHandler())
 
+	port := os.Getenv("SERVER_PORT")
+	if port == "" {
+		log.Fatalln("SERVER_PORT must be set in the ENV")
+	}
```

Comment:
IMO it's a better experience to default to a common HTP port in the user space (e.g. 8000) rather than requiring an env var be set.